app/modules/app.js

```diff
@@ -0,0 +1,140 @@
+var app = angular.module('Andejobs', ['ngMaterial','ngRoute', 'ngStorage']);
+
+
+// app.controller('AndejobsCtrl', ['$scope', function($scope){
```

Comment:
It's usually a good idea to delete code you no longer use i.e commented out code.